packaging/datadog-agent/source/supervisor.conf

```diff
@@ -19,7 +19,6 @@ serverurl = unix://%(here)s/../run/agent-supervisor.sock
 
 [program:collector]
 command=python agent/agent.py foreground --use-local-forwarder
-stdout_logfile=supervisord/logs/collector.log
```

Comment:
Yeah we should handle this case. 
We should sed the file regardless of it was existing or not i think.